Components/ListSearchResults/Partials/Pagination/_style.scss

```diff
@@ -1,29 +1,38 @@
 .pagination {
-  display: block;
-  margin-top: 40px;
+  margin-top: 20px;
 
-  div {
-    display: block;
-    margin: 10px 2px;
+  // Float clearfix
+  &::after {
+    clear: both;
+    content: '';
+    display: table;
+  }
+
+  &-nav {
+    width: 100%;
```

Comment:
I would rather stick to regular button widths and always float the buttons. This will also simply the css on this one.

Reply:
There is a high probability that on `320px` screen width, buttons won't have enough space to float as suggested, if different content is used (screenshot).

![no-space](https://user-images.githubusercontent.com/49150551/61525240-467c5080-aa18-11e9-90cd-30f07a5b1050.png)

Reply:
We might use the smaller button style instead.